presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -617,6 +617,35 @@ public static Slice rightPad(@SqlType("varchar(x)") Slice text, @SqlType(Standar
         return pad(text, targetLength, padString, text.length());
     }
 
+    @Description("computes edit distance between two strings")
+    @ScalarFunction
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long minDist(@SqlType("varchar(x)") Slice string1, @SqlType("varchar(y)") Slice string2)
```

Comment:
There exist various edit distance definitions (with or without substitution, with or without transposition) and best if users didn't have to guess. Including "levenshtein" in the function name can give them a hint.